src/push/fcm.rs

```diff
@@ -59,7 +59,7 @@ pub struct MessageResponse {
 /// FCM push result, sent inside the push response.
 #[derive(Debug, Deserialize)]
 pub struct MessageResult {
-    pub message_id: String,
+    pub message_id: Option<String>,
```

Comment:
This was the reason why parsing failed: The message id is not set for non-success responses.